testfiles/negative/systemd-networkd-wait-online-invalid-default-timeout.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+OnFailure=systemd-reboot.service
+Description=Wait for Network to be Configured
+Documentation=man:systemd-networkd-wait-online.service(8)
+DefaultDependencies=no
+Conflicts=shutdown.target
+Requires=systemd-networkd.service
+After=systemd-networkd.service
+Before=network-online.target shutdown.target
+
+[Service]
+Type=oneshot
+EnvironmentFile=-testfiles/negative/systemd-networkd-wait-online-invalid-timeout.env
+ExecStart=/bin/bash -c '/lib/systemd/systemd-networkd-wait-online ${OMNECT_WAIT_ONLINE_INTERFACES_RUN:---any --interface=eth0 --interface=wlan0 } --timeout=${OMNECT_WAIT_ONLINE_TIMEOUT_IN_SECS:-300}'
```

Comment:
`OMNECT_WAIT_ONLINE_INTERFACES_RUN` got renamed to
`OMNECT_WAIT_ONLINE_INTERFACES` and the default config is meanwhile only `--any`.
i guess this is not really relevant for the test, but if at some point somebody wonders.